### ISSUE TITLE
Abnormal saccade and pursuit detection

### ISSUE THREAD

**Issue by ruw001** [psychoinformatics-de/remodnav#43]
Hi,

Thanks for developing this great tool. It saved me a ton of time. I used a tobii 120hz eye tracker, and conducted a series of reading tasks where we track people's eyes when they read static passage. Most of the eye-movement events look reasonable, but for some participants, there are many pursuits which I think are falsely classified. I looked at the data, it seems some small saccades are not detected in the first phase (see below), so they are included inside fixation/pursuit. Is there any way to work around that? The only parameter I changed is `savgol-length`. I change it to 0.025 so it fits my sampling rate.
<img width="1000" alt="Screen Shot 2022-08-31 at 11 56 51 AM" src="https://user-images.githubusercontent.com/56607671/187736695-a87feafe-83e7-4f4b-bb65-8b0a4b4dd26d.png">

Another miss classification is the saccade onset (below). It looks like the adjacent small saccades are not detected successfully, thus the major saccade duration is very long. What should i do?
<img width="1000" alt="Screen Shot 2022-08-31 at 12 00 18 PM" src="https://user-images.githubusercontent.com/56607671/187737151-274b7e70-a876-4cfc-a183-255e5cb8afe1.png">

Thanks,
Ru

**Comment by adswa** [comment]
I don't have experience with reading data myself. I guess the saccades are smaller than during free viewing paradigms? I wonder if the parameters ``--min-saccade_duration`` and ``--min-pursuit_duration`` could be of help to allow shorter saccades and disallow short pursuits, and whether the default value of 2Hz for ``--max-initial-saccade-freq`` holds for reading tasks

**Comment by ruw001** [comment]
Hi Adina,

Thanks for your reply. I tweaked different parameters regarding saccade detection, and found that the saccade detection looks great when I set `max_initial_saccade_freq` to 4, with other parameters unchanged. 

For some participants with low calibration accuracy, their data might be noisier, so super long saccade duration still happens. I can work around by further setting `noise_factor` to 10.
